base/task.jl

```diff
@@ -13,6 +13,23 @@ function show(io::IO, t::Task)
     end
 end
 
+function copy(t::Task)
+  t.state != :runnable && t.state != :done &&  error("Only runnable or finished tasks can be copied.")
+  newt = ccall(:jl_copy_task, Any, (Any), t)::Task
+  if t.storage != nothing
+    newt.storage = copy(t.storage)
+  else
+    newt.storage = nothing
+  end
+  newt.code  = t.code
+  newt.result = t.result
+  newt.parent = t.parent
+  newt.last   = t.last
+  newt.consumers = jl_nothing;
+  newt.donenotify = jl_nothing;
+  newt
```

Comment:
fwiw, recompiling the c code is much faster than recompiling the sysimg (especially since `task.c` is not one of the cpp files that takes half a minute to compile). Of course a stand along julia script could be even faster.